### ISSUE TITLE
Booster age change Spanish translation

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#51188]
## Issue Description

We updated content on our Booster shots page about the age that boosters are suggested for. We need to get this text translated and update our Spanish page.

Word doc sent to translators: [https://app.zenhub.com/files/133843125/3492ad4a-d231-48da-a755-0e58abcaade3/download](https://app.zenhub.com/files/133843125/3492ad4a-d231-48da-a755-0e58abcaade3/download)

## Tasks
- [x] Send text to translators (Randi)
- [x] Update Spanish page (Aliyah)
- [x] Review (Megan)
- [x] Publish (Megan)

## Acceptance Criteria
- [x] Spanish content is updated on live site

**Comment by RLHecht** [comment]
@aliyahblackmore Here's the Spanish translation for you.[https://app.zenhub.com/files/133843125/9eed388b-e6b3-41c4-9188-2e8ba5516fba/download](https://app.zenhub.com/files/133843125/9eed388b-e6b3-41c4-9188-2e8ba5516fba/download)

**Comment by aliyahblackmore** [comment]
Hi @megzehn this is ready for you! 
**Preview link:** http://preview-prod.vfs.va.gov/preview?nodeId=34408

**Notes:**

- It seems like the translator decided to write out CDC. This was previously just the acronym. I'm adding in what we got from the translator. There's only one mention of "CDC" on this page, so I think this works!
- I also removed "seis" here. Wondering if there is a reason why we'd need to leave in "seis," but wanted to be consistent with mentions of numbers across the page.
